account_invoice_mass_sending/README.rst

```diff
@@ -13,17 +13,14 @@ Account Invoice Mass Sending
 .. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
-.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--invoicing-lightgray.png?logo=github
-    :target: https://github.com/OCA/account-invoicing/tree/14.0/account_invoice_mass_sending
-    :alt: OCA/account-invoicing
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount_invoicing-lightgray.png?logo=github
+    :target: https://github.com/OCA/account_invoicing/tree/14.0/account_invoice_mass_sending
```

Comment:
This change (across whole file) is not correct. Probably a wrong argument on the command line.